src/screens/SearchScreen.jsx

```diff
@@ -14,78 +14,80 @@ class SearchScreen extends Component {
     results: [],
     filterResults: [],
     loading: true,
-    deceased: false,
-    member: false,
+    showDeceased: false,
+    showExMember: false,
     searchText: '',
   };
 
   async componentDidMount() {
     const status = await Network.getNetworkStateAsync();
     if (status.isConnected) {
-      axios.get(`${i18n.locale}/api/v1/persons?fields=all&key=${Constants.manifest.extra.secretKey}`).then((res) => {
-        if (res.status == 200) {
-          this.setState({ results: res.data.result, loading: false });
-        }
-      }).catch(err => {
-        this.setState({ snackMsg: i18n.t('GENERAL.ERROR'), visible: true, loading: false })
-      });
+      axios
+        .get(`${i18n.locale}/api/v1/persons?fields=all`)
+        .then((res) => {
+          if (res.status == 200) {
+            this.setState({ results: res.data.result, loading: false });
+          }
+        })
+        .catch((err) => {
+          this.setState({ snackMsg: i18n.t('GENERAL.ERROR'), visible: true, loading: false });
+        });
     } else {
-      this.setState({ snackMsg: i18n.t('GENERAL.NO_INTERNET'), visible: true, loading: false })
+      this.setState({ snackMsg: i18n.t('GENERAL.NO_INTERNET'), visible: true, loading: false });
     }
-
   }
 
+  onChangeFilter = (texto) => {
+    let filterResults = [];
+
+    if (this.state.showDeceased == true && this.state.showExMember == true) {
```

Comment:
Las comparaciones `bool == true` son redundantes, pone nomas `this.state.showDeceased`, etc..